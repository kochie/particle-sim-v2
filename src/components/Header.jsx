import React from 'react';

export default class Header extends React.component {
	render() {
		return (
			<div>
			</div>
		)
	}
}
