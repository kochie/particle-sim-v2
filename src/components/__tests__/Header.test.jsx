import renderer from 'react-test-renderer';
import React from 'react';

import Header from '../Header';

function init(props) {
  const instance = renderer.create(<Header {...props} />);
  return { instance };
}

describe('header react component', () => {
  it('should render and match snapshot', () => {
    const { instance } = init();
    expect(instance).toMatchSnapshot();
  });
});
