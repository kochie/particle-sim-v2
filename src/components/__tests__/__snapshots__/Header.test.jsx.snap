// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header react component should render and match snapshot 1`] = `<div />`;
